WordPress/Classes/ViewRelated/Themes/ThemeBrowserCell.swift

```diff
@@ -193,10 +193,20 @@ open class ThemeBrowserCell: UICollectionViewCell {
         activityView.stopAnimating()
     }
 
-    fileprivate func refreshScreenshotImage(_ imageUrl: String) {
+    private func getImageUrlForWidth(imageUrl: String) -> String {
```

Comment:
NitPick: Style nitpick but in Swift, we usually leave off the `get` keyword for getters. 
```suggestion
    private func imageUrlForWidth(imageUrl: String) -> String {
```

Reply:
update method name to not have get